manifests/init.pp

```diff
@@ -61,7 +61,27 @@
   Optional[Integer] $logconfig_file_max_log_size = undef,
   Optional[Integer] $logconfig_file_backups      = undef,
   Optional[String] $logconfig_file_category      = undef,
+
+  # Padoptions
+  Etherpad::Stpadoptions $padoptions = {},
+
 ) {
+  #Default parameters for padoptions
+  $default_padoptions = {
+    noColors         => false,
+    showControls     => true,
+    showChat         => true,
+    showLineNumbers  => true,
+    useMonospaceFont => false,
+    userName         => false,
+    userColor        => false,
+    rtl              => false,
+    alwaysShowChat   => false,
+    chatAndUsers     => false,
+    lang             => 'en-gb',
+  }
+  #Merged values provides by user and default values
+    $_real_padoptions = merge($etherpad::default_padoptions, $etherpad::padoptions)
```

Comment:
Could you fix the indenting? You can also use `$default_padoptions` and `$padoptions` since they are both in local scope.

Reply:
Fixed.